datadog/resource_datadog_dashboard.go

```diff
@@ -540,6 +540,11 @@ func getNonGroupWidgetSchema() map[string]*schema.Schema {
 				Schema: getWidgetLayoutSchema(),
 			},
 		},
+		"id": {
+			Type:        schema.TypeInt,
```

Comment:
We've had some trouble in the past with widget IDs big enough to require Int64 when represented in Go. I think `schema.TypeInt` will work (they don't seem to provide an explicitly `Int64` version) but I'm not familiar enough to gauge this properly.

Just wanted to put this on your radar.

Reply:
Thanks for the comment: terraform does indeed the right thing, and TypeInt is int64 capable.